app/helpers/images_helper.rb

```diff
@@ -7,6 +7,6 @@ def display_image(name: "", path: "", extra_params: {})
     path = possible_paths.find{ |path| resolve_asset_path(path, true) }
     params = { alt: name.strip.titleize.capitalize }
     params.merge! extra_params
-    image_tag(path, params) if path
+    image_tag(root_url + image_path(path), params) if path
```

Comment:
pourquoi ne pas utiliser directement `image_url` ? 
https://apidock.com/rails/ActionView/Helpers/AssetUrlHelper/image_url

Reply:
Quitte, d'ailleurs, à faire une méthode de display_image spéciale pour les mails, pour garder la souplesse des `_path` dans l'appli

Reply:
Je ne savais pas qu'il y avait un parametre`host`

Reply:
Je me demande s'il manque pas aussi un peu de config genre https://guides.rubyonrails.org/action_mailer_basics.html#adding-images-in-action-mailer-views
mais bizarrement pour le moment ça fait pas ce que je veux dans les preview

Reply:
j'ai fait quelques changements

Reply:
ah c'est vrai qu'il y a une config pour le mailer je regarde

Reply:
apparement tout ce que j'ai fait ne sert a rien il suffit d'ajouter la config